### ISSUE TITLE
Needs access to admin database

### ISSUE THREAD

**Issue by s7dhansh** [adtribute/pub-sub-lite#5]
Is there a way the package would work without it?
I saw that it uses the admin access to check for version and replicaSet availability. Maybe we can add an explicit flag to skip those checks.

**Comment by npvn** [comment]
Yes the db admin access is currently only used for checking the compatibility of the db. If package users can guarantee that themselves then we won't need to perform this check.

I will add a flag for disabling this in a future version. Thanks for the suggestion!
